src/components/Divider/Divider.scss

```
@@ -0,0 +1,15 @@
+@use '../variables';
+
+$block: '.#{variables.$ns}divider';
+
+#{$block} {
+    &_orientation {
+        &_vertical {
+            border-inline-start: 1px solid var(--g-color-line-generic);
```

Let's add css api like this:
```suggestion
            border-inline-start: 1px solid var(--g-divider-color, var(--g-color-line-generic));
```

and add info about it to readme. https://github.com/gravity-ui/uikit/tree/main/src/components/Button#css-api